src/pe_reports/stakeholder/views.py

```diff
@@ -0,0 +1,595 @@
+import requests
+
+# cisagov Libraries
+from pe_reports.data.config import config
+from pe_reports.stakeholder.forms import InfoFormExternal
+
+logging.basicConfig(
+    filemode="a",
+    format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S",
+    level=logging.INFO,
+)
+
+# CSG credentials
+API_Client_ID = os.getenv("CSGUSER")
+API_Client_secret = os.environ.get("CSGSECRET")
+API_WHOIS = os.environ.get("WHOIS_VAR")
+
+conn = None
+cursor = None
+thedateToday = date.today().strftime("%Y-%m-%d")
+
+
+def getToken():
+    """Will get authorization token from CSG."""
+    d = {
+        "grant_type": "client_credentials",
+        "client_id": f"{API_Client_ID}",
+        "client_secret": f"{API_Client_secret}",
+    }
+    r = requests.post("https://api.cybersixgill.com/auth/token", data=d)
+    r = r.text.split(":")
+    r = r[1].lstrip('"').rsplit('"')[0]
+    return r
+
+
+def getAgencies(org_name):
+    """Get all agency names from P&E database."""
+    global conn, cursor
+    resultDict = {}
+    try:
+        params = config()
+
+        conn = psycopg2.connect(**params)
+
+        if conn:
+            logging.info(
+                "There was a connection made to"
+                "the database and the query was executed. "
+            )
+
+            cursor = conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
+
+            query = "select organizations_uid,name from"
+            " organizations where name='{}';"
+
+            cursor.execute(query.format(org_name))
+
+            result = cursor.fetchall()
+
+            for row in result:
+                theOrgUUID = row[0]
+                theOrgName = row[1]
+
+                resultDict[f"{theOrgUUID}"] = f"{theOrgName}"
+            return resultDict
+
+    except (Exception, psycopg2.DatabaseError) as err:
+        logging.error("There was a problem logging into the psycopg database %s", err)
+    finally:
+        if conn is not None:
+            cursor.close()
+            conn.close()
+            logging.info("The connection/query was completed and closed.")
+
+            return resultDict
+
+
+def getRootID(org_UUID):
+    """Get all agency names from P&E database."""
+    global conn, cursor
+    resultDict = {}
+    try:
+        params = config()
+
+        conn = psycopg2.connect(**params)
+
+        if conn:
+            logging.info(
+                "There was a connection made to the database and the query was executed "
+            )
+
+            cursor = conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
+            query = "select root_domain_uid, organization_name from"
+            " root_domains where organizations_uid='{}';"
+
+            cursor.execute(query.format(org_UUID))
+
+            result = cursor.fetchall()
+
+            for row in result:
+                theRootUUID = row[0]
+                theOrgName = row[1]
+
+                resultDict[f"{theOrgName}"] = f"{theRootUUID}"
```

Comment:
Same as above:
```suggestion
                # row[0] = root UUID
                # row[1] = org name
                resultDict[f"{row[1]}"] = f"{row[0]}"
```

Also, is there a reason why the UUID is the value in this `resultDict`, but it is the key in the previous (`getAgencies()`) one?  Just seems a little confusing to me.